doc/languages-frameworks/gradle.section.md

```diff
@@ -163,7 +163,7 @@ The Gradle setup hook accepts the following environment variables:
   the package if `doCheck` is set to `true`. Defaults to `test`.
 - `gradleUpdateTask` - the Gradle task (or tasks) to be used for
```

Comment:
This being named `gradleUpdateTask` makes no sense.

Reply:
Why? Because the command line does not explicitly run a Gradle task? FWIW Gradle implicitly `runs` help.

Do you have a proposal for a better name?

Reply:
I'm confused by your comment. This is the task that users can specify in order to capture dependencies. Why do you think `gradleUpdateTask` is not appropriate anymore?

Reply:
Because `--write-verification-metadata` isn't a task, it's a CLI flag

Reply:
@chayleaf @kira-bruneau I'd like to clarify the behavior of `--write-dependency-metadata`. It will resolve any dependencies, no matter which tasks are requested on the command line. That's why you usually run it without specifying a task, which will then run the default task, which is `help` if you've not configured to be something else. I'm strongly against running `$gradleBuildTask` and/or `$gradleCheckTask` because it will take way longer.

I think it makes sense to make sure it runs help, in case a project has the default task configured to something expensive. If we change the default of `gradleUpdateTask` to `help --write-dependency-metadata sha256` it makes sense again to call it `gradleUpdateTask`.

Reply:
Ok thanks for clearing that up! I think it's fair to mark this as resolved with the explicit help task included.